AxonDeepSeg/download_model.py

```diff
@@ -4,24 +4,27 @@
 
 
 def download_model(destination = None):
+    sem_release_version = 'r20211111v2'
```

Comment:
I installed the v4 and ran `axondeepseg_test` successfully.
However, the data test image packaged with the model in `r20211111v2` is the "old" one and does not correspond to the test image in the current main branch of [default_SEM_model](https://github.com/axondeepseg/default-SEM-model).

Reply:
Hey @mathieuboudreau,
I think the `r20211111v3` was made from master instead of the `mb/v4` branch, it contains the ADS model.
The [data_test](https://github.com/axondeepseg/default-SEM-model/tree/mb/v4/model_seg_rat_axon-myelin_sem/data_test) files on `mb/v4` branch are still the old ones.

Reply:
*facepalm* thanks! I'll do it asap.

Reply:
It *should* be correct now (I hope).

Reply:
Yes, all good! Thanks!